content/md/en/docs/main-docs/test/benchmark.md

```diff
@@ -170,7 +170,7 @@ For example:
     --extrinsic '\*' \
     --steps 20 \
     --repeat 10 \
-    --raw \
+    --json \
```

Comment:
```suggestion
```
This can just be removed now as a sane default. The human readable summary will still be printed. 

If the user wants the only JSON output printed he can use `--json` or if he wants it in a file he can use `--json-file`.  
Sorry for not making this more obvious in the original change.